connectionclass/src/main/java/com/facebook/network/connectionclass/DeviceBandwidthSampler.java

```diff
@@ -113,7 +113,7 @@ public void handleMessage(Message msg) {
      * Method for polling for the change in total bytes since last update and
      * adding it to the BandwidthManager.
      */
-    private void addSample() {
+    private void addSample(boolean cleanPreviousReadBytes) {
```

Comment:
I agree this would make it more readable. Update the pull request with the recommended change.